frontend/src/components/TrashPageLoggedinView.tsx

```diff
@@ -0,0 +1,121 @@
+import { useState, useEffect } from "react";
+import { Col, Row, Spinner, Modal, Button } from "react-bootstrap";
+import { DeletedNote as DeletedNoteModel } from "../models/deletedNote";
+import * as NotesApi from "../network/notes_api";
+import styles from "../styles/TrashPage.module.css";
+import DeletedNote from "./DeletedNote";
+import useTrashCountStore from "../store/trashCountStore";
+
+const TrashPageLoggedinView = () => {
+  const [deletedNotes, setDeletedNotes] = useState<DeletedNoteModel[]>([]);
+  const [deletedNotesLoading, setDeletedNotesLoading] = useState(true);
+  const [showNotesLoadingError, setShowNotesLoadingError] = useState(false);
+
+  const [showDeleteModal, setShowDeleteModal] = useState(false);
+  const [noteToDelete, setNoteToDelete] = useState<DeletedNoteModel | null>(
+    null
+  );
+
+  const { decrement } = useTrashCountStore();
+
+  useEffect(() => {
+    async function loadNotes() {
+      try {
+        setShowNotesLoadingError(false);
+        setDeletedNotesLoading(true);
+        const deletedNotes = await NotesApi.fetcDeletedNotes();
+        setDeletedNotes(deletedNotes);
```

Comment:
don't forget to set the loading to false and error to false after validating you have the deleted notes